planet_explorer/tests/test_daily_imagery.py

```diff
@@ -391,3 +396,429 @@ def test_search_wrong_aoi(qtbot, logged_in_explorer_dock_widget, qgis_debug_enab
     qtbot.keyClicks(dock_widget._aoi_filter.leAOI, "wrong AOI")
     qgis_debug_wait(qtbot, qgis_debug_enabled)
     qtbot.mouseClick(dock_widget.btnSearch, QtCore.Qt.LeftButton)
+
+
+@pytest.mark.parametrize(
+    "name, layer_dir, expected_coordinates",
+    [
+        pytest.param(
+            "polygons",
+            "/usr/src/planet_explorer/tests/data/aoi_tests/test_aoi.gpkg",
```

Comment:
Can we use a relative path here and elsewhere?

Reply:
All paths are now relative